### ISSUE TITLE
Show current speed limit if car navigation is active

### ISSUE THREAD

**Issue by FranzSchi** [mapsme/omim#3543]
The current speed limit administrated for the current used road should be displayed in the GUI in case that car navigation is active.
See also my FB Question here:
https://www.facebook.com/photo.php?fbid=10206202092403015

**Comment by OSMmapper** [comment]
It would be very nice if the maxspeed was displayed in navigation mode.
But dont forget to include the "maxspeed:conditional" too.

**Comment by my-na-me** [comment]
And of course, once you add this, please don't forget parameters like maxspeed:forward=* / maxspeed:backward=* ;-)

**Comment by codingABI** [comment]
Yes, showing the current speed limit would be great and is working on apps like Magic Earth

**Comment by ktaragorn** [comment]
Is there a particular reason this is not being worked on/implemented? I might try my hand at it unless there is a particular reason not to.
